server/utils/nunjucksSetup.ts

```diff
@@ -38,6 +40,11 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
       express: app,
     },
   )
+
+  njkEnv.addGlobal('t', (key: string) => {
+    return i18next.t(key)
+  })
+
```

Comment:
So the `t` function does indeed work without this, however removing this method causes the template rendering tests to fail, which I believe is because they now rely on the `t` global helper being available